pvl/decoder.py

```diff
@@ -109,6 +137,14 @@ def raise_unexpected(self, stream, token=None):
     def raise_unexpected_eof(self, stream):
         self.raise_error('Unexpected EOF', stream)
 
+    def broken_parameter_value(self, lineno):
+        if self.strict:
+            msg = "Broken Parameter-Value. Set strict to False to skip error"
```

Comment:
After (possibly) addressing the naming issue above, consider rewording this message to be more explicit.  Your message tells the user what to do to make this work, but could be reworded so that they know exactly what to do.  Try:

"Broken Parameter-Value.  Using 'strict=False' when calling 'pvl.load' may help you parse the label, it could also inadvertently mask other errors."

The "inadvertently masking errors" bit is the warning to the user that using the strict option might not be a great idea.  Which it may not be.  It's our disclaimer.